api/wizards.yaml

```diff
@@ -0,0 +1,22 @@
+openapi: 3.0.0
+components:
+  schemas:
+    Wizard:
+      type: object
+      properties:
+        steps:
+          type: array
+          items:
+            $ref: "#/components/schemas/WizardStep"
```

Comment:
It is stronger than me:
![image](https://github.com/kubeshop/tracetest/assets/12397879/168e41ba-e024-4bfe-b9c3-3cb0f6011ff6)

https://www.youtube.com/watch?v=Sagg08DrO5U

Reply:
😂